starlette/datastructures.py

```diff
@@ -382,7 +382,7 @@ def update(
         self._dict.update(value)
 
 
-class QueryParams(ImmutableMultiDict):
+class QueryParams(ImmutableMultiDict[str, str]):
```

Comment:
Just to make MyPy happy

Reply:
Also means we now get `QueryParams(...).get(...)` to be a `str`, rather than `Any`. Which is correct, right?

Reply:
Yep! There's a lot of downstream work from this (like actually fixing `Form`), but I wanted to keep this PR as narrowly scoped as possible. Hopefully we can approve and merge those downstream PRs quicker once the basic structure for stronger typing is added via this PR.